compiler/src/dotty/tools/dotc/transform/ReifyQuotes.scala

```diff
@@ -157,6 +157,8 @@ class ReifyQuotes extends MacroTransform {
                 val tagDef = tagDefCache.getOrElseUpdate(prefix.symbol, mkTagSymbolAndAssignType(prefix))
                 tagDef.symbol.typeRef
             }
+          case AnnotatedType(parent, _) =>
+            apply(parent) // Only keep the Annotated tree
```

Comment:
Does it mean it throws the annotation away? That will make it impossible to keep some annotations in types.

Reply:
We throw away the annotation on the type but then recover it from the `Annotated` type tree while unpickling. 
Maybe I should only do it if it contains a splice. In that case we would always have the tree.